airgap_instructions.adoc

```diff
@@ -0,0 +1,206 @@
+CaaSP is sourced from three types of repositories:
+
+. RPM packages - MicroOS
+.. Software updates on the operating system
+.. Provided by SUSE package repositories
+. Container images - Docker, CRI-O
```

Comment:
Do we have to mention docker and cri-o other there? It kinda gives the impression we have two kind of container images based on the engine being used. I would just drop that or add something like "used by the container engine"